saleor/plugins/base_plugin.py

```diff
@@ -137,7 +137,7 @@ def __str__(self):
     #  Authenticate user which should be assigned to the request.
     #
     #  Overwrite this method if the plugin handles authentication flow.
-    authenticate_user: Callable[[WSGIRequest], Union["User", NoneType]]
+    authenticate_user: Callable[[WSGIRequest, Any], Union["User", NoneType]]
```

Comment:
Any?

Reply:
All definitions of methods in BasePlugin have `previous_value` declared as `Any`. I added it here also as it was missing previously.

Reply:
I don't think `Any` is the correct type in any of those cases? Isn't previous value always the same type as returned value?